src/System.Memory/tests/Performance/Perf.ReadOnlySequence.TryGet.cs

```diff
@@ -14,7 +14,7 @@ public class Perf_ReadOnlySequence_TryGet
         private const int InnerCount = 10_000;
         volatile static int _volatileInt = 0;
 
-        [Benchmark(InnerIterationCount = InnerCount)]
+        [Benchmark(InnerIterationCount = InnerCount * 10)]
```

Comment:
Same here, just bump InnerCount to 100_000 instead of having `* 10` everywhere.

Reply:
For example, I can remove multiplication in all tests and add division in MultiSegments tests.

Reply:
Yes.